src/coreclr/vm/jitinterface.h

```diff
@@ -109,6 +109,12 @@ BOOL LoadDynamicInfoEntry(Module *currentModule,
 // The portable helper is used if the platform does not provide optimized implementation.
 //
 
+#ifdef JIT_PollGC
+EXTERN_C FCDECL0(void, JIT_PollGC);
+#else
+#define JIT_PollGC NULL
+#endif
```

Comment:
I assume this has something to do with the defining of `JIT_PollGC` to the same name. We should avoid this pattern in the final PR and come up with a better mechanism.